config/certmanager/certificate.yaml

```diff
@@ -0,0 +1,39 @@
+# The following manifests contain a self-signed issuer CR and a certificate CR.
+# More document can be found at https://docs.cert-manager.io
+# WARNING: Targets CertManager v1.0. Check https://cert-manager.io/docs/installation/upgrading/ for breaking changes.
+apiVersion: cert-manager.io/v1
```

Comment:
do I get it correct that with this cert-manager is a requirement and needs to be installed?

Reply:
It just seems to work without needing any explicit install, as far as I have seen in my testing

Reply:
> +1 from me. @dprince and @fmount, what do you think?

In general I'm ok with the patch,  but given we're in the development phase, something like:

`ENABLE_WEBHOOKS=false GOWORK= OPERATOR_TEMPLATES=./templates make run`

doesn't work anymore because we need to fix first the `create_self_signed_cert.sh`: I'll comment on the script.

`make run` works as expected, but I'd like to avoid breaking the good old way of running the operators locally :D

Reply:
> do I get it correct that with this cert-manager is a requirement and needs to be installed?

Sorry, I missed to answer this question earlier. Though cert-manager scaffolding is created by sdk, cert-manager does not inject the certs atm. webhook uses the same OLM generated certificate as api service.

```
       volumeMounts:
        - mountPath: /tmp/k8s-webhook-server/serving-certs
          name: webhook-cert
          readOnly: true
        - mountPath: /apiserver.local.config/certificates
          name: apiservice-cert
      dnsPolicy: ClusterFirst
      restartPolicy: Always
      schedulerName: default-scheduler
      securityContext:
        runAsNonRoot: true
      serviceAccount: keystone-operator-controller-manager
      serviceAccountName: keystone-operator-controller-manager
      terminationGracePeriodSeconds: 10
      volumes:
      - name: webhook-cert
        secret:
          defaultMode: 420
          items:
          - key: tls.crt
            path: tls.crt
          - key: tls.key
            path: tls.key
          secretName: keystone-operator-controller-manager-service-cert
      - name: apiservice-cert
        secret:
          defaultMode: 420
          items:
          - key: tls.crt
            path: apiserver.crt
          - key: tls.key
            path: apiserver.key
          secretName: keystone-operator-controller-manager-service-cert

```